pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -29,6 +29,8 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+const ConditionCleanedUp apis.ConditionType = "CleanedUp"
```

Comment:
NIT: `CleanedUp` does not resonate with me very much as a description for this condition. 

BTW I was working on doing something similar for the initialization steps, i.e. setting start time,  setting the initializing the `Succeeded` condition, the cloud events status. I wanted to store the specs in the status so that we only fetch / validate them once and then mark the initialization as successful.

So perhaps we could have three conditions:
- `PreRunSucceeded` (before steps for Task and before DAG for PipelineRuns)
- `Succeeded`
- `PostRunSucceeded` (after steps for Task and after DAG for PipelineRuns)